includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -174,7 +170,51 @@ private function check_stable_tag( Check_Result $result, array $files ) {
 				$result,
 				__( 'The Stable Tag in your readme file does not match the version in your main plugin file.', 'plugin-check' ),
 				'stable_tag_mismatch',
-				$file
+				$readme_file
+			);
+		}
+	}
+
+	/**
+	 * Checks the readme file warnings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Check_Result $result The Check Result to amend.
+	 * @param string       $readme_file Readme file.
+	 * @param Parser       $parser The Parser object.
+	 */
+	private function check_for_warnings( Check_Result $result, string $readme_file, Parser $parser ) {
+		$warnings = $parser->warnings;
+
+		$warning_keys = array_keys( $warnings );
+
+		$ignored_warnings = array(
+			'contributor_ignored',
+		);
+
+		/**
+		 * Filter the list of ignored readme parser warnings.
+		 *
+		 * @since n.e.x.t
+		 *
+		 * @param array  $ignored_warnings Array of ignored warning keys.
+		 * @param Parser $parser The Parser object.
+		 */
+		$ignored_warnings = (array) apply_filters( 'plugin_check_readme_warnings_ignored', $ignored_warnings, $parser );
```

Comment:
```suggestion
		$ignored_warnings = (array) apply_filters( 'wp_plugin_check_readme_warnings_ignored', $ignored_warnings, $parser );
```

We use `wp_` in filters for Plugin Check.

Could you please add tests coverage for the filter?